compose/aspnet-mssql-compose.md

```diff
@@ -29,25 +29,28 @@ configure this app to use our SQL Server database, and then create a
     [Docker Desktop for Mac](/docker-for-mac/#/file-sharing), you
     need to set up file sharing for the volume that you need to map.
 
-1.  Within your directory, use the `dotnet:2.1-sdk` Docker image to generate a
+1.  Within your directory, use the `mcr.microsoft.com/dotnet/core/sdk:3.1` Docker image to generate a
     sample web application within the container under the `/app` directory and
     into your host machine in the working directory:
 
     ```bash
-    $ docker run -v ${PWD}:/app --workdir /app microsoft/dotnet:2.1-sdk dotnet new mvc --auth Individual
+    $ docker run --rm -v ${PWD}:/app --workdir /app mcr.microsoft.com/dotnet/core/sdk:3.1 dotnet new mvc --auth Individual
     ```
 
     > **Note**: If running in Docker Desktop for Windows, make sure to use Powershell
     or specify the absolute path of your app directory.
 
 1.  Create a `Dockerfile` within your app directory and add the following content:
 
-    ```conf
-    FROM microsoft/dotnet:2.1-sdk
+    ```dockerfile
+    FROM mcr.microsoft.com/dotnet/core/sdk:3.1
+    RUN ["dotnet", "tool", "install", "--global", "dotnet-ef"]
+    ENV PATH "~/.dotnet/tools/:$PATH"
     COPY . /app
     WORKDIR /app
     RUN ["dotnet", "restore"]
```

Comment:
The pattern used here is very different than in the [asp.net sample](engine/examples/dotnetcore.md).  I realize this PR is about upgrading to 3.1 but it is something that jumped out at me.

Reply:
That's basically because the EF migrations are being used when the container is run.  It's very much an "interactive development" kind of sample.  Since this is just about getting the existing samples in a working state on 3.1, I'm not going to concern myself with this aspect of the sample at this time.